src/ontology/util/OntologyUtils.java

```diff
@@ -52,6 +62,8 @@ private OntologyUtils(ProcessingEnvironment processingEnv) {
 
         // Built-in ontic concepts for isomorphic types.
         LIST = elements.getTypeElement("java.util.List").asType();
+        DICTIONARY = elements.getTypeElement("java.util.Dictionary").asType();
+        MAP = elements.getTypeElement("java.util.Map").asType();
     }
 
     public static void initOntologyUtils (ProcessingEnvironment processingEnv) {
```

Comment:
File an issue to refactor the hacky intialization process.

Reply:
https://github.com/opprop/units-inference/blob/master/src/units/representation/UnitsRepresentationUtils.java#L127 might be useful to copy